riscv/insns/gorci.h

```diff
@@ -1,8 +1,12 @@
 // Zbb contains orc.b but not general gorci
-if (SHAMT == 7)
-  require_extension(EXT_ZBB);
-else
+try {
+  if (SHAMT == 7) {
+    require_extension(EXT_ZBB);
+  }
+  else throw trap_illegal_instruction(0);
+} catch(...) {
   require_extension(EXT_XBITMANIP);
+}
```

Comment:
This looks correct but rather convoluted.  I think we want:

    require((p->extension_enabled(EXT_ZBB) && (SHAMT == 7)) || p->extension_enabled(EXT_XBITMANIP));